charts/jaeger/values.yaml

```diff
@@ -96,6 +81,75 @@ allInOne:
     fsGroup: 10001
   securityContext: {}
 
+service:
```

Comment:
I don't like how you are mixing the OTEL config sections with chart sections like `provisionDataStore`. Is it possible to wrap the config under parent `config:` element?